spec/book_spec.rb

```diff
@@ -0,0 +1,24 @@
+require './spec/spec_helper'
+describe Book do
+  before :all do
+    @book = Book.new('Literature', 'good', 12 / 10 / 2022)
```

Comment:
[Optional] Kindly wrap the date in a single quote. maybe you mean?
`@book = Book.new('Literature', 'good', '12/10/2022')`